src/components/Settings.js

```diff
@@ -70,6 +92,71 @@ class Settings extends Component {
           />
           <span>Oversample</span>
         </div>
+
+        <hr />
+
+        <div>Top Bound</div>
+        <input
+          className={classNames('Settings-input', {
+            'Settings-input-error': !isPositiveInteger(width)
```

Comment:
Your inputs have logic attached to them that indicates they should get error states, but that doesn't seem to be the case. Look at the existing Settings inputs for this behavior.

Reply:
Done